infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/provision/secret/GitCredentialStorageFileSecretApplier.java

```diff
@@ -68,7 +69,10 @@ public void applySecret(KubernetesEnvironment env, RuntimeIdentity runtimeIdenti
       if (gitConfig != null) {
         if (gitConfig.contains("helper = store --file") && gitConfig.contains("[credential]")) {
           throw new InfrastructureException(
-              "Multiple git credentials secrets found. Please remove duplication.");
+              format(
+                  "Multiple git credential secrets for user %s found in namespace %s. That may be caused by reinstalling product without user namespaces cleanup or using multiple instances of product with the same namespace namings template.",
```

Comment:
can we add the exact names of the secrets?

Reply:
not easy cuz we processing all kinds of secrets in NS ant call appropriate applier with only given secret.